spec/activity_stream_spec.rb

```diff
@@ -0,0 +1,9 @@
+describe AnsibleTowerClient::ActivityStream do
+  let(:url)                 { "example.com/api/v1/activity_stream" }
+  let(:api)                 { AnsibleTowerClient::Api.new(instance_double("Faraday::Connection")) }
+  let(:collection)          { api.activity_stream }
+  let(:raw_url_collection)  { build(:response_url_collection, :klass => described_class, :url => url) }
+  let(:raw_instance)        { build(:response_instance, :group, :klass => described_class) }
+
+  include_examples "Collection Methods"
```

Comment:
https://github.com/ansible/ansible_tower_client_ruby/blob/master/spec/support/shared_examples/crud_methods.rb only has an example for 'POST' 

Do you want shared examples for read in this PR? I'd rather like to do it in follow up, as I need this to move further for the event catcher in miq